blocks/ads-block/themes/commerce.json

```diff
@@ -19,12 +19,15 @@
 		"styles": {
 			"default": {
 				"background-color": "$global.neutral-7",
-				"padding": "$global.spacing-4",
+				"padding-block-start": "$global.spacing-4",
+				"padding-inline-end": "$global.spacing-4",
+				"padding-block-end": "$global.spacing-4",
+				"padding-inline-start": "$global.spacing-4",
 				"border-radius": "var(--border-radius)",
```

Comment:
@blakeganderson This will need to be converted to logical properties as well. See https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values#properties_for_border_radius.